src/main/scala/org/broadinstitute/dsde/agora/server/ServerInitializer.scala

```diff
@@ -32,8 +36,18 @@ class ServerInitializer(val config: Config) extends LazyLogging {
   }
 
   private def startWebServiceActors() {
+    implicit val timeout = Timeout(5.seconds)
```

Comment:
The ask requires it so I figured I'd keep it as close to this usage as possible. should it be up at the beginning of the class?

5 seconds because that's what Doug had. Better value?